react/components/slider/index.js

```diff
@@ -2,10 +2,10 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import Slider from 'react-slick'
 
-import Dots from './Dots'
-import Arrow from './Arrow'
+import Dots from './components/Dots'
+import Arrow from './components/Arrow'
 
-import getItemsPerPage from './getItemsPerPage'
+import getItemsPerPage from './components/ItemsPerPage'
```

Comment:
ItemsPerPage is not a component, it's just a function :thinking: 

I think that is better to put this file in the component root './' ?

Reply:
Make senses.